### ISSUE TITLE
Not working with lock hooked with HF-JA2

### ISSUE THREAD

**Issue by btcdtc** [banban525/echonetlite2mqtt#36]
I have a lock hooked with HF-JA2 adapter, it works in aiseg controller but I found I am unable to control this lock when I tried this plugin.

I found there is OperationStatus under NodeProfile 
![Screenshot 2024-11-09 at 22 02 43](https://github.com/user-attachments/assets/19bde9f7-d940-446d-bd8c-043e01951880)
but no OperationStatus node under switch,
![Screenshot 2024-11-09 at 22 04 53](https://github.com/user-attachments/assets/f7f6b16e-66f8-4cb0-bcb7-42691e77033c)
the spec looks correct but I tried lock and unlock yet the every nodes stays the same.
![Screenshot 2024-11-09 at 22 15 12](https://github.com/user-attachments/assets/817f7ed3-244f-4e0c-92a9-e7de54d6e243)

Could you help to provide the REST API url so I can check what api it provides? Thanks.

update:
I used picoGW and this is what I got. and all data stays the same regardless the lock status.
![Screenshot 2024-11-09 at 23 34 17](https://github.com/user-attachments/assets/3fb740c7-0255-47bf-bf9f-b7554efa571b)

**Comment by banban525** [comment]
@btcdtc 
Thank you for your report.

Based on my diagnosis from the screenshot, ECHONETLite2MQTT is not able to collect OperationalStatus from the device.
Does restarting ECHONETLite2MQTT not improve this problem?

So far I don't know what the cause is.
If possible, could you please provide the detailed log at the bottom of the LOG page?

> Could you help to provide the REST API url so I can check what api it provides? Thanks.

I'm not sure what you need for this.
The device REST API provided by ECHONETLite2MQTT is as follows.

/elapi/v1/devices/(device id)
/elapi/v1/devices/(Device Id)/properties

However, the REST API has the same information as the WEB page.

**Comment by btcdtc** [comment]
@banban525 

Thanks for you response,

I  deployed many echonetlite tools and none of them able to collect or set OperationalStatus.
the current ECHONETLite2MQTT and  picoGW are deployed on raspberry pi with only Ethernet connect in the same 192.168.20.0/24 subnet. 
 
![Screenshot 2024-11-11 at 0 11 53](https://github.com/user-attachments/assets/df015411-ac15-4588-b31b-76d1797d24a9)

seems none of the tools can successfully get a response if they request OperationalStatus, So my guess is maybe since it is already linked to my aiseg console, it will reject all other request. I will try to unlink it and try again tomorrow.

![Screenshot 2024-11-11 at 0 12 16](https://github.com/user-attachments/assets/b5a28637-baaf-4929-8259-52062dba67a7)
![Screenshot 2024-11-11 at 0 13 05](https://github.com/user-attachments/assets/7f684666-303e-4146-9158-6cd5f2d53d2c)

**Comment by btcdtc** [comment]
I found the rootcause.
I delete it from aiseg and found aiseg cannot no longer pair with it succfully.
and then I found when paried with a lock, HF-JA2 will enable 1:1 communication mode by default. 

https://av.jpn.support.panasonic.com/support/share2/eww/jp/hnavi/manual_JA2_AiSEG.pdf

and I can't found how to turn it off.
https://av.jpn.support.panasonic.com/support/share2/eww/jp/hnavi/hf-ja2_basic.pdf

**Comment by banban525** [comment]
@btcdtc 

Thanks for reporting.

The following URL reports similar behavior, but for a different device:

https://x.com/Ranas_Genoa/status/1815672718299144580?t=F6mOmYxAU9TOTwrzVxVPYg&s=19

The problematic device may only be able to connect to either the aiseg2 or the echonetlite

**Comment by btcdtc** [comment]
I fixed the issue,
the reason is the HF-JA2 adapter has been set to 1:1 mode. HF-JA2 has a simple management webui http://ip:8080
but there isn't much can be done, the pair mode is not there. 
so I reset the adapter and  I confirmed it can be controlled. late when I tried hook it with Aiseg and I realized at the end of the setting it asks if this device is Aiseg only or Aiseg and home navigator.  I assuemd this means 1:1 mode or 1:many mode and choose the later which is indeed 1:many mode. now I can control the lock through different management tool.

and thanks for your work on this, it is a life saver.
